presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaAdminFactory.java

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Rename commit message to `Add predicate push down support for internal Kafka columns`. The reminder can be phrased as:
```
Add support for predicate pushdown for following Kafka internal columns
 * _timestamp 
 * _partition_offset
 * _partition_id

If predicate specifies lower bound on _timestamp column (_timestamp > XXXX), it is always pushed down. 
The upper bound predicate is pushed down only if allowed via ``kafka.timestamp-upper-bound-force-push-down-enabled`` config property
or ``timestamp_upper_bound_force_push_down_enabled`` session property.
```

```